src/foam/u2/wizard/controllers/IncrementalWizardController.js

```diff
@@ -178,6 +178,7 @@ foam.CLASS({
             for ( let w of this.data.wizardlets ) {
               if ( w.submit ) w.submit();
             }
+            this.onClose(x, true);
```

Comment:
```suggestion
```

This one isn't needed - handled by `postSet` in `data` property

Reply:
tested - and yes you are correct